.travis/install.sh

```diff
@@ -11,9 +11,8 @@ then
 
     # Update brew itself
     export HOMEBREW_NO_AUTO_UPDATE=1  # Auto-updating everything would take too much time
-    brew update
     brew cleanup  # Preempt possible scheduled clean-up so it doesn't clutter the log later
-
+    brew update > /dev/null
```

Comment:
does it have a special reason that you moved brew update to below brew cleanup?

Reply:
in general, this PR looks good to merge now, this is the only question left.

the timeout issues still happening for macosx are likely unrelated to this PR.

Reply:
yes, my fault! re-located in the correct place